src/ruby.rs

```diff
@@ -1187,6 +1347,13 @@ tests! {
         assert.rb_eq(lazy_eval("::Math::DomainError"), unsafe { rb_eMathDomainError });
     }
 
+    #[test]
+    fn test_types(assert: &mut Assertions) {
+        assert.rs_eq(unsafe { RB_T_STRING }, unsafe { T_STRING });
+        assert.rs_ne(unsafe { RB_TYPE_P("foo".to_ruby(), T_STRING) }, 0);
```

Comment:
I was confused for a second since I didn't notice it switched from `eq` to `ne`. We should probably just have a `fn c_bool(value: c_int) -> bool {}` function in the `testing` module and write this as `rs_eq(c_bool(...), true)`

Reply:
Seems ok.